_posts/commands/2020-05-01-mom_saveloc_current.md

```diff
@@ -7,4 +7,4 @@ tags:
 safeguard: mom_run_safeguard_saveloc_tele
 ---
 
-Teleports the player to their current saved location.
\ No newline at end of file
+Teleports the player to their current saved location. If player holds down the key it will load the saveloc and freeze the player in place, releasing the key will unfreeze the player.
```

Comment:
This behaviour works only if `+mom_saveloc_current` is bound, not just `mom_saveloc_current`. (`+mom_saveloc_current` loads the saveloc and freezes in place, and `-mom_saveloc_current` unfreezes player).